app/controllers/home_controller.rb

```diff
@@ -47,6 +47,29 @@ def dashboard
     end
   end
 
+  def dashboard2
```

Comment:
Can we have better name `dashboard2 `?

Reply:
Yeah, I can use v2 which sounds much better...